src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Resource/NewAzureResourceCmdlet.cs

```diff
@@ -63,9 +63,10 @@ public sealed class NewAzureResourceCmdlet : ResourceManipulationCmdletBase
         /// <summary>  
         /// Gets or sets the plan object.  
         /// </summary>  
+        [Alias("SkuObject")]  
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "A hash table which represents sku properties.")]  
         [ValidateNotNullOrEmpty]  
-        public Hashtable SkuObject { get; set; }  
+        public Hashtable Sku { get; set; }  
```

Comment:
We should also update help with an example with this parameter name change and add unit test.